app/helpers/fhir_helper.rb

```diff
@@ -232,6 +270,8 @@ def from_us_core_ethnicity(patient)
     code = patient&.extension&.select { |e| e.url.include?(url) }&.first&.extension&.select { |e| e.url == 'ombCategory' }&.first&.valueCoding&.code
     return 'Hispanic or Latino' if code == '2135-2'
     return 'Not Hispanic or Latino' if code == '2186-5'
+    return 'Unknown' if code == 'UNK'
+    return 'Refused to Answer' if code == 'ASKU'
```

Comment:
This also needs to be updated per the corresponding changes you made in the `to_us_core_ethnicity` method.

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/767b8765f1830cb0e34e8bc8b1adbbe48c0786f5#diff-402d451d1f7019322100522f6898c3153444edb8ddb963e181a457c84a2ca6b6R277